source/globalize-fetch.js

```diff
@@ -6,9 +6,9 @@ function status(response) {
   if (response.status >= 200 && response.status < 300) {
     return response
   } else {
-    let error = new Error(response.statusText)
+    let error = new Error(response.statusText);
     // attach the original response to the thrown error
-    ;(error: any).response = response
+    (error: any).response = response
```

Comment:
I revisit this whenever we change flow versions… still haven't come up with a better solution.